packages/react/src/components/Masthead/Masthead.js

```diff
@@ -245,6 +245,7 @@ const Masthead = ({
                 title={title}
                 eyebrowText={eyebrowText}
                 eyebrowLink={eyebrowLink}
+                navigation={mastheadData}
```

Comment:
Make this pass in L1 specific navigation data instead, maybe `navigationL1`?

Reply:
Okay, that makes sense.